TestCases/compliance-level-3/0084-feel-for-loops/0084-feel-for-loops.dmn

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0084-feel-for-loops" name="0084-feel-for-loops" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <description>FEEL for loops</description>
+
+    <decision name="decision_001" id="_decision_001">
+        <variable name="decision_001"/>
+        <literalExpression>
+            <text>for i in [1,2,3] return i + 1</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_002" id="_decision_002">
+        <variable name="decision_002"/>
+        <literalExpression>
+            <text>for i in [1,2,3], j in [4,5] return i + j</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003" id="_decision_003">
+        <variable name="decision_003"/>
+        <literalExpression>
+            <text>for i in [2..4] return i</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003_a" id="_decision_003_a">
+        <variable name="decision_003_a"/>
+        <literalExpression>
+            <text>for i in [4..2] return i</text>
```

Comment:
analogously, this should be rewritten as:
```
for i in 4..2 return i
```

Reply:
as per above